recipes/radio_beam/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "radio_beam" %}
+{% set version = "0.2" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "3b3b3cfcc17d61df469c9e45d7f7fc993b902f3f2c076279071fd984a6e5763d" %}
+
+package:
+  name: radio-beam
```

Comment:
This is deliberate - the package name on PyPI will change in future, so just pre-empting this to avoid changes in conda